lib/improver/cli/neighbour_finding.py

```diff
@@ -276,8 +285,18 @@ def process(orography, landmask, site_list, metadata_dict=None,
     # Deal with coordinate systems for sites other than PlateCarree.
     if 'site_coordinate_system' in kwargs.keys():
         scrs = kwargs['site_coordinate_system']
-        kwargs['site_coordinate_system'] = safe_eval(scrs, ccrs,
-                                                     PROJECTION_LIST)
+        if scrs not in PROJECTION_LIST:
+            raise ValueError('invalid projection {}'.format(scrs))
+        site_crs = getattr(ccrs, scrs)
+        scrs_opts = json.loads(kwargs['site_coordinate_options'])
+        if 'globe' in scrs_opts:
+            crs_globe = ccrs.Globe(**scrs_opts['globe'])
+            del scrs_opts['globe']
+        else:
+            crs_globe = ccrs.Globe()
+        kwargs['site_coordinate_system'] = site_crs(
+            globe=crs_globe, **scrs_opts)
+        del kwargs['site_coordinate_options']
```

Comment:
Is there a reason you didn't `pop` this on L291? (I can understand the explicit `del scts_opts['globe']`)

Reply:
No reason other than not thinking of it when I was writing the code. I've changed this to use `pop` as it's a neater way to write the same thing.